lib.rs

```diff
@@ -238,6 +241,18 @@ impl<T: ?Sized> Arc<T> {
     }
 }
 
+// `no_std`-compatible abort by forcing a panic while already panicing.
+fn abort() -> ! {
```

Comment:
We should use a cfg for this

Reply:
That's because this is based on top of the other branch. Annoying it shows up...